### DISCUSSION TITLE
Customize Users DISPLAYED COLUMNS

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5606] [Q&A] [answered]
In CP, how to customize DISPLAYED COLUMNS in users view table, currently it shows email, name, role and last login. I updated the blueprint and added new field call Type, how to show it in the table that display the results ?

**Comment by HazemMohamed252** [comment]
Any one to answer ?

**Comment by jackmcdade** [comment, marked as answer]
At this time there is no way to customize the user column fields. It's on our todo list.

**Comment by bdedardel** [comment]
Hi, is it still in the todo list or is it implemented ?

**Reply by duncanmcclean** [reply]
This is possible in Statamic 6, which is currently in alpha.

**Reply by bdedardel** [reply]
Thanks for your response, i will upgrade (and thanks for the migration guide : https://runway.duncanmcclean.com/upgrade-guides/v5-x-to-v6-0)
